tests/test_interface_ospf.rb

```diff
@@ -406,13 +406,13 @@ def multiple_interface_config_hash
         interfaces[0].downcase => {
           area: '0.0.0.0', cost: 10, hello: 30, dead: 120, pass: true },
         interfaces[1].downcase => {
-          area: '1.1.1.38', dead: 40, pass: false },
+          area: '1.1.1.38', cost: 10, dead: 40, pass: false },
```

Comment:
Are you sure you need these changes.  I believe that https://github.com/cisco/cisco-network-node-utils/commit/c8e649aff87302058a473fd92d43b09667f43a46#diff-e736b9d63e07f75b6647c6cbe0838ccaR392 may have fixed this issue.